modules/developer-lightspeed/con-about-lightspeed-stack-and-llama-stack.adoc

```diff
@@ -0,0 +1,19 @@
+:_mod-docs-content-type: CONCEPT
+
+[id="con-about-lightspeed-stack-and-llama-stack_{context}"]
+= About {lcs-name} and Llama Stack
+
```

Comment:
In the preview build I noticed  {rhdh-short} appears in the output.  Looks like  {rhdh-short} has to be added to the attributes file.